File: src/go/dry.go
package main

import "fmt"

func DoStuff() {
	fmt.Println("Doing stuff!")
}

func DoSum(a int, b int) int {
	return (a + b)
}

func main() {
	DoStuff()
	fmt.Printf("2 + 1 = %v", DoSum(2, 1))
}

File: src/kt/dry.kt
fun doStuff() {
    println("Doing stuff!")
}

fun doSum(a: Int, b: Int) :Int {
	return a + b
}

fun main(args: Array<String>) {
    doStuff()
    print("2 + 1 = ${doSum(2, 1)}")
}